### ISSUE TITLE
Some issues in conv_transpose2d. 

### ISSUE THREAD

**Issue by dhiyu** [pytorch/vision#2205]
Recently I met this problem bothering me.
In TensorFlow, there is a funciton:
`tf.nn.conv2d_transpose(
    input, filters, output_shape, strides, padding='SAME', data_format='NHWC',
    dilations=None, name=None
)`
But in PyTorch:
`torch.nn.functional.conv_transpose2d(input, weight, bias=None, stride=1, padding=0, output_padding=0, groups=1, dilation=1) → Tensor`
As you can see there is no parameter named output_shape. But my project need this parameter to recitify the size of output. Anyone can help me? Thanks~

**Comment by pmeier** [comment]
> recitify the size of output

What does that mean? Could you give an example how you want the output size to be rectified?

---

In general, `conv_transpose2d` has no parameter `output_shape` since it is completely determined by the other parameters.

- The batch size, i. e. `output.size()[0]`, does not change 
- The number of channels of the output, i. e. `output.size()[1]`, is determined by `weights.size()[1]`
- The size of spatial dimensions of the output, i. e. output.size()[2:], are determined according to [these formulas](https://pytorch.org/docs/stable/nn.html#torch.nn.ConvTranspose2d) in the subsection `Shape`.

**Comment by fmassa** [comment]
Hi,

We use the issues for tracking bugs and feature requests.

This question is unrelated to torchvision, and is better suited for the forums http://discuss.pytorch.org/ 

As such, I'm closing this issue.
